EEJT-0002/src/main/java/com/epam/esm/exceptionhandler/exception/ServerException.java

```diff
@@ -0,0 +1,7 @@
+package com.epam.esm.exceptionhandler.exception;
+
+public class ServerException extends ApplicationException {
```

Comment:
What is the difference between `ApplicationException` and  `ServerException`? When do use one or another? Please provide examples

Reply:
i remake this , it is just copypaste